plugins/formal-verification/formver.viper/src/org/jetbrains/kotlin/formver/viper/Verifier.kt

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2010-2023 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.formver.viper
+
+import org.jetbrains.kotlin.formver.viper.ast.Program
+import viper.silicon.Config
+import viper.silicon.logger.MemberSymbExLogger
+import viper.silicon.logger.`NoopSymbExLog$`
+import viper.silicon.logger.SymbExLogger
+import viper.silicon.verifier.DefaultMainVerifier
+import viper.silver.cfg.silver.SilverCfg
+import viper.silver.reporter.StdIOReporter
+
+sealed interface VerifierError {
+    val msg: String
+}
+
+data class ConsistencyError(override val msg: String) : VerifierError
+data class VerificationError(override val msg: String) : VerifierError
+
+class Verifier {
+    private val verifier: DefaultMainVerifier
+
+    init {
+        val config = Config(seqOf("--ignoreFile", "dummy.vpr"))
```

Comment:
Given that any non-empty config requires these flags to be present, I'd prefer to keep them there so that we don't have to rediscover that when we want to add another option.

Reply:
Hmm that's interesting that those flags are needed. In that case we should keep them.